cocos/core/assets/texture-cube.jsb.ts

```diff
@@ -84,43 +84,13 @@ textureCubeProto._ctor = function () {
     this._mipmapAtlas = null;
 };
 
-function _forEachFace (mipmap, callback: (face, faceIndex) => void) {
-    callback(mipmap.front, FaceIndex.front);
-    callback(mipmap.back, FaceIndex.back);
-    callback(mipmap.left, FaceIndex.left);
-    callback(mipmap.right, FaceIndex.right);
-    callback(mipmap.top, FaceIndex.top);
-    callback(mipmap.bottom, FaceIndex.bottom);
-}
-
 Object.defineProperty(textureCubeProto, 'mipmaps', {
     get () {
         return this._mipmaps;
     },
     set (value) {
         this._mipmaps = value;
         this.setMipmaps(value);
-        this.mipmapLevel = this._mipmaps.length;
-        if (this._mipmaps.length > 0) {
-            const imageAsset = this._mipmaps[0].front;
-            this.reset({
-                width: imageAsset.width,
-                height: imageAsset.height,
-                format: imageAsset.format,
-                mipmapLevel: this._mipmaps.length,
-            });
-            this._mipmaps.forEach((mipmap, level) => {
-                _forEachFace(mipmap, (face, faceIndex) => {
-                    this.assignImage(face, level, faceIndex);
-                });
-            });
-        } else {
-            this.reset({
-                width: 0,
-                height: 0,
-                mipmapLevel: this._mipmaps.length,
```

Comment:
Don't need to invoke reset & forEach logic in ts since it is done in cpp by `setMipmaps JSB call` .